packages/katello/rubygem-pulp_deb_client/rubygem-pulp_deb_client.spec

```diff
@@ -0,0 +1,96 @@
+# template: scl
+%{?scl:%scl_package rubygem-%{gem_name}}
+%{!?scl:%global pkg_name %{name}}
+
+%global gem_name pulp_deb_client
+%global gem_require_name %{gem_name}
+
+%global release 1
+%global prerelease b1
+%global prereleaserpm %{?prerelease:.}%{?prerelease}
+
+Name: %{?scl_prefix}rubygem-%{gem_name}
+Version: 2.3.0
+
+Release: %{?prereleaserpm:0.}%{release}%{?prereleaserpm}%{?dist}
+Summary: Pulp 3 DEB plugin API Ruby Gem
+Group: Development/Languages
+License: GPL-2.0+
+URL: https://github.com/pulp/pulp_deb/
+Source0: https://rubygems.org/gems/%{gem_name}-%{version}.gem
```

Comment:
I think its failing because you have to add the prerelease here:

```Source0: https://rubygems.org/gems/%{gem_name}-%{version}%{?prerelease}.gem```